process/README.md

```diff
@@ -3,63 +3,55 @@
-
+Unlike many checks, the Process check doesn't monitor anything useful by default. You must configure which processes you want to monitor.
 
-1. While there's no standard default check configuration, here's an example `process.d/conf.yaml` that monitors SSH/SSHD processes. See the [sample process.d/conf.yaml][3] for all available configuration options:
+While there's no standard default check configuration, here's an example `process.d/conf.yaml` that monitors SSH/SSHD processes. See the [sample process.d/conf.yaml][3] for all available configuration options:
 
 ```yaml
-  init_config:
-
-  instances:
-
-  ## @param name - string - required
-  ## Used to uniquely identify your metrics as they are tagged with this name in Datadog.
-  #
-        - name: ssh
-
-  ## @param search_string - list of strings - optional
-  ## If one of the elements in the list matches, it returns the count of
-  ## all the processes that match the string exactly by default. Change this behavior with the
-  ## parameter `exact_match: false`.
-  ##
-  ## Note: Exactly one of search_string, pid or pid_file must be specified per instance.
-  #
-          search_string:
-            - ssh
-            - sshd
+init_config:
+instances:
+- name: ssh
+  search_string:
+    - ssh
+    - sshd
 ```
 
-Some process metrics require either running the Datadog collector as the same user as the monitored process or privileged access to be retrieved. Where the former option is not desired, and to avoid running the Datadog collector as `root`, the `try_sudo` option lets the Process Check try using `sudo` to collect this metric. As of now, only the `open_file_descriptors` metric on Unix platforms is taking advantage of this setting. Note: the appropriate sudoers rules have to be configured for this to work:
+**Note**: After you make configuration changes, make sure you [restart the Agent][4].
 
-   ```text
-   dd-agent ALL=NOPASSWD: /bin/ls /proc/*/fd/
-   ```
+Retrieving some process metrics requires either running the Datadog collector as the same user as the monitored process, or using privileged access. For the `open_file_descriptors` metric on Unix platforms, there is an additional configuration option. Setting `try_sudo` to `true` in your `conf.yaml` file allows the Process check to try using `sudo` to collect the `open_file_descriptors` metric. Using this configuration option requires setting the appropriate sudoers rules in `/etc/sudoers`:
 
-2. [Restart the Agent][4].
+```shell
+dd-agent ALL=NOPASSWD: /bin/ls /proc/*/fd/
+```
 
 ### Validation
 
 Run the [Agent's status subcommand][5] and look for `process` under the Checks section.
 
 ### Metrics notes
 
-**Note**: Some metrics are not available on Linux or OSX:
+The following metrics are not available on Linux or OSX:
+- Process I/O metrics are **not** available on Linux or OSX since the files that the Agent reads (`/proc//io`) are only readable by the process's owner. For more information, [read the Agent FAQ][6].
```

Comment:
```suggestion
- Process I/O metrics are **not** available on Linux or macOS since the files that the Agent reads (`/proc//io`) are only readable by the process's owner. For more information, [read the Agent FAQ][6].
```